source/module_io/write_input.cpp

```diff
@@ -108,6 +108,11 @@ void Input::Print(const std::string &fn) const
     {
         ModuleBase::GlobalFunc::OUTP(ofs, "pw_diag_ndim", pw_diag_ndim, "max dimension for davidson");
     }
+    else if (ks_solver == "dav_subspace")
+    {
+        ModuleBase::GlobalFunc::OUTP(ofs, "pw_diag_ndim", pw_diag_ndim, "dimension of workspace (number of wavefunction packets, at least 2 needed)");
+        ModuleBase::GlobalFunc::OUTP(ofs, "diago_full_acc", pw_diag_ndim, "if all the empty states are diagonalized at the same level of accuracy of the occupied ones.");
+    }
     ModuleBase::GlobalFunc::OUTP(ofs,
```

Comment:
Please add the description of diago_full_acc in docs/advanced/input_files/input-main.md

Reply:
I have added it in the latest commit.